plugins/techdocs/src/home/components/TechDocsPageWrapper.tsx

```diff
@@ -34,16 +37,18 @@ export type TechDocsPageWrapperProps = {
  * @public
  */
 export const TechDocsPageWrapper = (props: TechDocsPageWrapperProps) => {
-  const { children } = props;
+  const { children, title, subtitle, showSubtitle = true } = props;
   const configApi = useApi(configApiRef);
-  const generatedSubtitle = `Documentation available in ${
-    configApi.getOptionalString('organization.name') ?? 'Backstage'
-  }`;
+  const generatedSubtitle =
+    subtitle ||
+    `Documentation available in ${
+      configApi.getOptionalString('organization.name') ?? 'Backstage'
+    }`;
 
   return (
     <PageWithHeader
-      title="Documentation"
-      subtitle={generatedSubtitle}
+      title={title || 'Documentation'}
+      subtitle={showSubtitle ? generatedSubtitle : undefined}
```

Comment:
With these props, could we also follow the approach I mentioned above, where we explicitly pass the react components we want to render as props, but just keep the existing defaults when those props aren't passed in?